extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -23,7 +23,9 @@ export const MicrosoftDatatoolsSchemaSqlSql = 'Microsoft.Data.Tools.Schema.Sql.S
 export const databaseSchemaProvider = 'DatabaseSchemaProvider';
 
 // Project Provider
-export const projectTypeDisplayName = localize('projectTypeDisplayName', 'Database Project');
+export const sqlDatabaseProjectTypeId = 'sqldbproj';
+export const projectTypeDisplayName = localize('projectTypeDisplayName', "Database Project");
```

Comment:
Should this be SQL Database Project to clarify?

Reply:
more information is available in description